How to use Modernizr v3?

Trying to use Modernizr3 and unfortunately I can't really figure it out. Installing from `npm` does not work:

```js
var modnernizr = require('modernizr')
```

Cloning and installing the repo, then running `./bin/modernizr` leads me to the following error:

```sh
{ [AssertionError: missing path]
  name: 'AssertionError',
  actual: undefined,
  expected: true,
  operator: '==',
  message: 'missing path',
  generatedMessage: false }
```

--- a/bin/modernizr
+++ b/bin/modernizr
@@ -150,10 +150,14 @@ if (argv.c) {
 }
 
 try {
-  if (!inlineConfig) {
+  if (!inlineConfig && configPath) {
     config = require(configPath);
   } else if (configPath) {
     config = _.extend(config, require(configPath));
+  } else if (!configPath) {
+    console.error('config file, inline features, or options required.');
+    yargs.showHelp();
+    process.exit(1)
   }
 } catch (e) {
   console.error(configPath + ' is not valid JSON.');

